scipy/io/matlab/tests/test_mio.py

```diff
@@ -1195,5 +1195,21 @@ def test_empty_mat_error():
     assert_raises(MatReadError, loadmat, sio)
 
 
+def test_unicode_utf16():
+    # Test we save unicode data as utf16, not utf8
+    # See https://github.com/scipy/scipy/issues/4431
+    sio = BytesIO()
+    savemat(sio, {'char': u'Python'})
```

Comment:
This isn't valid syntax on Python 3.2, which is why the tests failed. Use ``u()`` from ``scipy._lib.six``.